data/sql/updates/pending_db_world/rev_1621437037316104400.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1621437037316104400');
+
+DELETE FROM `gossip_menu_option` WHERE `MenuID`=941 AND `OptionID`=1;
+DELETE FROM `gossip_menu_option_locale` WHERE `MenuID`=941 AND `OptionID`=1;
+DELETE FROM `broadcast_text` WHERE `ID`=4165;
```

Comment:
Dont delete broadcast_text ever

Reply:
My apologies, I changed it and now I know for the future